### PR TITLE
Revise block explorer routing to meet EIP-3091 standards

### PR DESCRIPTION
**Pull request by Ayagoumi**
Updates the block explorer routing system to align with the EIP-3091 standard for Ethereum-based BEs. The new structure and syntax are: 

- `/tx/:hash` for transaction detail
- `/block/:id` for block detail
- `/txs` for all transaction
- `/blocks` for all blocks

### CODE REVIEW

File: src/app/pages/CChainPages/Blocks/BlockDetails.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { useLocation } from 'react-router-dom'
 import { useAppDispatch, useAppSelector } from 'store/configureStore'
 import { fetchCBlockDetail } from 'store/cchainSlice/utils'
 import { getCBlockDetail, getCBlockDetailStatus } from 'store/cchainSlice'
-import { CCHAIN, CTRANSACTIONS } from 'utils/route-paths'
+import { CCHAIN, CTRANSACTION } from 'utils/route-paths'
```
Comment (Chriserus) [inline src/app/pages/CChainPages/Blocks/BlockDetails.tsx]: I think CTRANSACTION isn't used here

File: src/app/pages/CChainPages/LatestBlocksAndTransactionsList/index.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { FC } from 'react'
 import { Grid } from '@mui/material'
 import { BlockTableData } from 'types/block'
 import { CTransaction } from 'types/transaction'
-import { CBLOCKS, CTRANSACTIONS } from 'utils/route-paths'
+import { CBLOCKS, CTRANSACTION } from 'utils/route-paths'
```
Comment (Chriserus) [inline src/app/pages/CChainPages/LatestBlocksAndTransactionsList/index.tsx]: Same here, not used anymore

File: src/app/pages/CChainPages/Blocks/BlockDetails.tsx (hunk 1 above)
Comment (Ayagoumi) [reply src/app/pages/CChainPages/Blocks/BlockDetails.tsx]: Done

File: src/app/pages/CChainPages/LatestBlocksAndTransactionsList/index.tsx (hunk 2 above)
Comment (Ayagoumi) [reply src/app/pages/CChainPages/LatestBlocksAndTransactionsList/index.tsx]: Done

**Review by Chriserus — DISMISSED** [review]

File: src/app/index.tsx [hunk 3]
```diff
@@ -29,30 +29,22 @@ export function App() {
...
                         <Route index element={<CChainPage />} />
-                        <Route path="blocks">
-                            <Route index element={<Blocks />} />
-                            <Route path=":id" element={<BlockDetails />} />
-                        </Route>
-                        <Route path="transactions">
-                            <Route index element={<CTransactions />} />
-                            <Route path=":id" element={<TransactionDetails />} />
-                        </Route>
+                        <Route path="blocks" element={<Blocks />} />
```
Comment (aeddaqqa) [inline src/app/index.tsx:32]: why not use variables from utils/route-paths.ts

**Review by aeddaqqa — DISMISSED** [review]
LGTM

**Review by Chriserus — APPROVED** [review]

**Review by aeddaqqa — APPROVED** [review]

**Pull request merged**